src/devices/quotra.ts

```diff
@@ -14,7 +22,9 @@ const definitions: Definition[] = [
         model: 'B07JHL6DRV',
         vendor: 'Quotra',
         description: 'RGB WW LED strip',
-        extend: [light({colorTemp: {range: [150, 500]}, color: true})],
```

Comment:
Use this instead: `extend: [light({colorTemp: {range: [150, 500]}, color: true, powerOnBehaviour: false})],`